arviz/plots/plot_utils.py

```diff
@@ -253,6 +256,14 @@ def purge_duplicates(list_in):
     return _list
 
 
+def _dims(data, var_name, skip_dims):
+    return [dim for dim in data[var_name].dims if dim not in skip_dims]
+
+
+def _zip_dims(new_dims, vals):
+    return [{k: v for k, v in zip(new_dims, prod)} for prod in product(*vals)]
```

Comment:
Have these functions been created for performance reasons? They are a single line of code and only called once

Reply:
I was aiming for a loop lift via numba but Sphinx was failing. I left these methos as such so as to jit them later with a later, compatible version of Numba. Numba-devs are planning for some major changes with their following versions so I am writing jitted functions separately for debugging.